tests/acceptance/selecting-package-type-test.js

```diff
@@ -0,0 +1,44 @@
+import Ember from 'ember';
+import { module, test } from 'qunit';
+import startApp from '../helpers/start-app';
+import '../factories/offer';
+import '../factories/item';
+import '../factories/package';
+import FactoryGuy from 'ember-data-factory-guy';
+import TestHelper from 'ember-data-factory-guy/factory-guy-test-helper';
+
+var App, offer, item1, item2, package1, package2, item2, packageType;
+
+module('Reviewer: Accept Item Tab', {
+  beforeEach: function() {
+    App = startApp({}, 2);
+    TestHelper.setup();
+
+    offer = FactoryGuy.make("offer", { state:"under_review"});
+    item1 = FactoryGuy.make("item_with_type", { state: "accepted"});
+    item2 = FactoryGuy.make("item", { offer: offer});
+    packageType = item1.get('packageType');
+    package1 = FactoryGuy.make("package", { item: item1, packageType: packageType});
+    package2 = FactoryGuy.make("package", { item: item1, packageType: packageType});
+
+    $.mockjax({url: '/api/v1/package_types', type: 'GET', status: 200, responseText: {
+        package_types: [packageType.toJSON({includeId: true}), packageType.toJSON({includeId: true})]
```

Comment:
What is the reason for returning same `packageType` twice?

Reply:
Just return two package_types which are same.